src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolCaseMappingTest.java

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
need to enable and fix all the drool unit tests

Reply:
done